core/web/evm_chains_controller_test.go

```diff
@@ -79,6 +81,7 @@ func Test_EVMChainsController_Show(t *testing.T) {
 					BlockHistoryEstimatorBlockHistorySize: null.IntFrom(50),
 					EvmEIP1559DynamicFees:                 null.BoolFrom(true),
 					MinIncomingConfirmations:              null.IntFrom(12),
+					LinkContractAddress:                   null.StringFrom(utils.ZeroAddress.String()),
```

Comment:
Let's use a random address here to make sure we get it right `cltest.NewAddress` outta do it

Reply:
Understood!